src/plots/plot.ts

```diff
@@ -11,32 +28,29 @@ namespace Plottable.Plots {
     scale?: Scale<D, R>;
   }
 
+  export interface TransformableAccessorScaleBinding<D, R> {
```

Comment:
Can you add some docs for this interface? It feels very important but I don't know what it's used for